bookmarks/api/routes.py

```diff
@@ -97,11 +97,14 @@ def unarchive(self, request, pk):
     @action(methods=["get"], detail=False)
     def check(self, request):
         url = request.GET.get("url")
+        disable_cache = request.GET.get("disable_cache") == "true"
```

Comment:
The query param doesn't really need a value, the presence of `disable_cache` could be enough.